### DISCUSSION TITLE
v1.4.0

### DISCUSSION THREAD

**Discussion by MrMeowzz** [MrMeowzz/Funkin-MrMeowzz#25] [Updates]
- added note types (tricky, bob, fnf hd)
- added new option for ratings to follow camera
- added new option for new hit timings

<hr /><em>This discussion was created from the release <a href='https://github.com/MrMeowzz/Funkin-MrMeowzz/releases/tag/v1.4.0'>v1.4.0</a>.</em>

**Comment by DeceptedPar** [comment]
nice!

**Reply by DeceptedPar** [reply]
in the debug menu when placing the fire notes with note hit sounds the notes aren't lined right

**Reply by DeceptedPar** [reply]
the pixel fire notes aren't alligned right

**Reply by DeceptedPar** [reply]
the sprite is one box above where it is, the hitbox is where it's supposed to be but the sprite is not
